app/scripts/models/user.js

```diff
@@ -524,6 +530,19 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    beginFlow: function (flowBeginTime) {
+      if (! this.has('flowId')) {
```

Comment:
Is `flowId` used for anything besides guarding whether `this._flow.begin` should be called? If not, does it make sense to move that guard logic into the flow model so all the logic is consolidated?